README.md

```diff
@@ -39,9 +60,54 @@ You can contribute by proposing new goals for a focus area, or new questions for
 or new metrics for answering those questions. Or by helping to refine goals, and questions.
 
 
-## Meetings
 
-We meet every Wednesday at 11am CDT (usually 18:00 CET, but beware different switches to Summer time in EU and US, [check your local time](http://www.thetimezoneconverter.com/?t=11am&tz=Chicago&)) in the CHAOSS Zoom room https://unomaha.zoom.us/j/720431288 (Meeting ID for [dial in](https://unomaha.zoom.us/zoomconference?m=DKGo2mmIuOv9xSjphoGZZmYKxr5HFrS9): 720 431 288)
+
+TODO: Fill out a short section that links to D&I metrics repo and explains how this repo fits into overall CHAOSS metrics effort.
+[D&I](https://github.com/chaoss/wg-diversity-inclusion)
+
+
+
+## Background
+* Background - A lot of the current content at the top of the repo will be moved here and expanded on.
+
+
+
+## Usage
```

Comment:
If we want to include structure, I would add the main directories, explaining the contents of each one of them. Maybe some single file, if relevant. But not all files, etc.: they can be already browsed in the repo itself, and we would have a lot of work maintaining this list. On the contrary, I see value in referencing each directory, with a short description of what is intended to be its conents. Something like:

*  focus_areas: description of all the focus areas of the working group, including the goals, questions and name of metrics (with links to specific metric definitions in the corresponding directory, see below).
* metrics: description of all the metrics used in the different  focus areas.
* implementations: reference implementations of the metrics
* use_cases: use cases relevant to the focus areas of the group

Reply:
I think this is still relevant. Please fix the text below accordingly.

Reply:
I will just put   "To be written ..." under *Usage
The reason being that, I don't yet have a full understanding of the GMD-wg and how things are structured.